enterprise/admin/views.py

```diff
@@ -90,6 +91,100 @@ def get_user_name(request):
         return request.user.first_name or request.user.username
 
 
+class EnterpriseCustomerTransmitCoursesView(View):
+    """
+    Transmit courses view.
+
+    Allows transmitting of courses metadata for provided enterprise.
+    """
+    template = 'enterprise/admin/transmit_courses_metadata.html'
+
+    class ContextParameters(object):
```

Comment:
why is it required to have inner class for context constants ? 

**suggestion** --- I think, you can remove the inner class and define in outer class or You can either move these constants to external class and access with class name directly if they can be used externally in some other methods in future. e.g. 

```
class ContextParameters(object):
         ENTERPRISE_CUSTOMER = 'enterprise_customer'

ContextParameters.ENTERPRISE_CUSTOMER
```

Reply:
These context parameters are only limited to this view.
Also this format is used in existing admin views.